### DISCUSSION TITLE
Still facing the same issue on PyInstaller

### DISCUSSION THREAD

**Discussion by mallinlr** [barrust/pyspellchecker#147] [General]
Hi friend,

I have tried with below command; 
pyinstaller --add-binary="spellchecker/resources/en.json.gz:spellchecker/resources" my_prog.py

But, still I am getting the same issue;

pyinstaller: error: argument --add-binary: invalid add_data_or_binary value: 'spellchecker/resources/en.json.gz:spellchecker/resources'

Please suggest to fix it.

Thank you

**Comment by h1ckstead** [comment]
@mallinlr did you change "spellchecker/resources/" to your actual path? In my case, it looks like this:
` --add-binary="C:\Users\MyUsername\PycharmProjects\MyProjectName\venv\Lib\site-packages\spellchecker\resources\en.json.gz;spellchecker/resources"`
Remember, if you're on windows to use `;` instead of `:`
